src/parse/validator.rs

```diff
@@ -166,25 +166,56 @@ impl<'b, 'c, 'z> Validator<'b, 'c, 'z> {
         Ok(())
     }
 
-    fn build_conflict_err(&self, name: Id, matcher: &ArgMatcher) -> ClapResult<()> {
-        debugln!("build_err!: name={}", name);
-        let usg = Usage::new(self.p).create_usage_with_title(&[]);
-        if let Some(other_arg) = self.p.app.find(name) {
-            for &k in matcher.arg_names() {
-                if let Some(a) = self.p.app.find(k) {
-                    if let Some(ref v) = a.blacklist {
-                        if v.contains(&name) {
-                            return Err(Error::argument_conflict(
-                                a,
-                                Some(other_arg.to_string()),
-                                &*usg,
-                                self.p.app.color(),
-                            ));
+    fn build_usg_message(&self, name: Id, matcher: &ArgMatcher) -> (Option<&Arg<'_>>, Vec<u64>) {
+        struct Cache<'a> {
+            arg: Option<&'a Arg<'a>>,
+            blacklist: Vec<u64>,
+        }
+        // Accumulate all blacklists and wanted arg
+        let result = matcher.arg_names().fold(
+            Cache {
+                arg: None,
+                blacklist: vec![],
+            },
+            |mut acc, key| {
+                if let Some(arg) = self.p.app.find(*key) {
+                    if let Some(ref v) = arg.blacklist {
+                        if v.contains(&name) && acc.arg.is_none() {
+                            acc.arg = Some(arg);
                         }
+                        acc.blacklist = [&acc.blacklist[..], &v].concat()
                     }
                 }
+                acc
+            },
+        );
+
+        // Get args which are not blacklisted
+        let allowed_args = matcher.arg_names().fold(vec![], |mut acc, key| {
+            if !result.blacklist.contains(&key) {
+                acc.push(*key);
+            }
+            acc
+        });
```

Comment:
updated. Thanks

Reply:
FYI, there's the `Resolve conversation` button here. You are supposed to push it instead of typing "updating". One click instead of 8 button types, let's be energy efficient.